src/bfrini.f90.in

```diff
@@ -0,0 +1,314 @@
+!> within internal modules and COMMON blocks throughout the BUFRLIB
+!> software.
+!>
+!> <p>This subroutine isn't normally called directly by any
+!> application program, because it's automatically called internally
+!> from within subroutine openbf() during the first time that subroutine
+!> is called by any application program.
+!>
+!> @authors J. Woollen
+!> @authors J. Ator
+!> @date 1994-01-06
+!>
+!> <b>Program history log:</b>
+!> - 1994-01-06  J. Woollen -- Original author
+!> - 1995-06-28  J. Woollen -- Increased the size of internal BUFR table
+!>                           arrays in order to handle bigger files
+!> - 1998-07-08  J. Woollen -- Modified to make Y2K-compliant
+!> - 1999-11-18  J. Woollen -- The number of BUFR files which can be
+!>                             opened at one time increased from 10 to 32
+!>                             (necessary in order to process multiple
+!>                             BUFR files under the MPI)
+!> - 2000-09-19  J. Woollen -- Maximum message length increased from
+!>                           10,000 to 20,000 bytes
+!> - 2003-11-04  J. Ator    -- Added documentation
+!> - 2003-11-04  S. Bender  -- Added remarks and routine interdependencies
+!> - 2003-11-04  D. Keyser  -- MAXJL (maximum number of jump/link entries)
+!>                           increased from 15000 to 16000 (was in
+!>                           verification version); initialize
+!>                           variable JSR to zero in new COMMON block
+!>                           /BUFRSR/ (was in verification version);
+!>                           unified/portable for WRF; added history
+!>                           documentation
+!> - 2004-08-18  J. Ator    -- Added initialization of COMMON /MSGSTD/;
+!>                           maximum message length increased from
+!>                           20,000 to 50,000 bytes
+!> - 2005-11-29  J. Ator    -- Added initialization of COMMON /MSGCMP/
+!>			    and calls to pkvs1 and pkvs01()
+!> - 2009-03-23  J. Ator    -- Added initialization of COMMON /DSCACH/,
+!>                           COMMON /MSTINF/ and COMMON /TNKRCP/
+!> - 2012-09-15  J. Woollen -- Modified for C/I/O/BUFR interface,
+!>                           added initialization of COMMON blocks
+!>                           /ENDORD/ and /BUFRBMISS/
+!> - 2014-09-15  J. Ator    -- Change default location of MTDIR
+!> - 2014-11-18  J. Ator    -- Added initialization of modules MSGLIM
+!>                           and USRINT; remove S01CM initialization
+!> - 2016-11-29  J. Ator    -- Explicitly initialize BMISS as 10E10_8
+!> - 2017-10-13  J. Ator    -- Added initialization of COMMON /TABLEF/
+!> - 2019-05-03  J. Ator    -- Change default location of MTDIR
+!> - 2019-05-09  J. Ator    -- Added dimensions for MSGLEN
+!> - 2021-03-23  J. Ator    -- Use mtinfo() with MASTER_TABLE_DIR
+!>                             macro from CMake
+!>
+      SUBROUTINE BFRINI
```

Comment:
since this file is now rendered as `.f90`, each line can start on the 1st space and there is no need for 6 spaces like the fixed format would require.

Reply:
Agreed, I'll go ahead and fix this in bfrini.f90 for consistency with the other .f90 source code.